Formula/tarantool.rb

```diff
@@ -47,6 +47,11 @@ def install
     args << "-DCURSES_NCURSES_LIBRARY=/usr/lib/libncurses.dylib"
     args << "-DICONV_INCLUDE_DIR=#{sdk}/usr/include"
 
+    system "xcrun", "--sdk", "macosx", "--show-sdk-path"
+    system "find", "#{sdk}", "-name", "libcurl.*", "2>/dev/null"
```

Comment:
I'd really rather you figure out how to build for macOS on your own CI runners. We only have 3 runners per macOS version and those aren't really meant for upstream to debug their build.

Reply:
Ok, I see. I'll try to find some hardware to check OSX 11.0. Anyway may be you have any suggestions how to fix our issue, we have some investigation in tarantool/tarantool#5543:
```
    The failure comes from tarantool/cmake/FindCURL.cmake:108-134
    in 18e2420. In fact, any check_c_source_runs() fail leads to
    the 'Curl was built without SSL support' failure. The fail
    looks so in CMakeError.txt or when --trace-expand option is
    passed to cmake:
    
      Run Build Command(s):/usr/bin/make cmTC_6cf00/fast && /Applications/Xcode_12.2.app/Contents/Developer/usr/bin/make  -f CMakeFiles/cmTC_6cf00.dir/build.make CMakeFiles/cmTC_6cf00.dir/build
      Building C object CMakeFiles/cmTC_6cf00.dir/src.c.o /Applications/Xcode_12.2.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DCURL_SUPPORTS_SSL -D_GNU_SOURCE  -Wno-unknown-pragmas -fexceptions -funwind-tables -fno-omit-frame-pointer -fno-stack-protector -fno-common -msse2  -isysroot /Applications/Xcode_12.2.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk -o CMakeFiles/cmTC_6cf00.dir/src.c.o -c /Users/runner/work/tarantool/tarantool/CMakeFiles/CMakeTmp/src.c
      make[2]: *** No rule to make target `/usr/lib/libcurl.dylib', needed by `cmTC_6cf00'.  Stop.
      make[1]: *** [cmTC_6cf00/fast] Error 2
    
    So the /usr/lib/libcurl.dylib file just does not exist.
    The path is passed from the brew formula using the
    '-DCURL_LIBRARY=/usr/lib/libcurl.dylib' cmake option.
    
    Guess about the reason of the failure that Mac OS 11.0
    (Big Sur) introduces the following:
    
      New in macOS Big Sur 11.0.1, the system ships with a
      built-in dynamic linker cache of all system-provided
      libraries. As part of this change, copies of dynamic
      libraries are no longer present on the filesystem.
      Code that attempts to check for dynamic library presence
      by looking for a file at a path or enumerating a
      directory will fail. Instead, check for library presence
      by attempting to dlopen() the path, which will correctly
      check for the library in the cache. (62986286)
    
    So, the /usr/lib/libcurl.dylib file does not present on the
    file system, but it seems check_c_source_runs() expects
    that it exists.
    
    We can expect that SDK or CLI Developer tools is present on
    a build system and link against SDK's libcurl. Even if an
    SDK will not present at the target system, the linker cache
    will allow a dynamic loader to see the libraries as if they
    would be in /usr/lib (I hope).
```

Reply:
I gave my suggestion in the linked issue. I already looked at your results and (in a private discussion) pointed that after this patch the passed option is `-DCURL_LIBRARY=/usr/lib/libcurl.tbd`, so `sdk` variable is empty. I suggested you to look how path to the SDK (or CLI developer tools) is filled in other formulae.

Please, share results of this step first, then ask for further help.

And I agree with @SMillerDev: there is no reason to do any PRs until you'll verify that everything works — locally or using GitHub Actions in your own repository.

Reply:
If you need the SDK path, I think you want `MacOS.sdk_path_if_needed`.

In any case, `sdk` is empty because of how it's defined. CLT is always installed on Homebrew CI.